colin/checks/result.py

```diff
@@ -39,20 +40,6 @@ def __init__(self, ok, description, message, reference_url, check_name, severity
         self.correction_diff = correction_diff
 
 
-class ContainerCheckResult(AbstractResult):
```

Comment:
Why do you remove it? Just a question. Would you like to have only one `CheckResult`?

Reply:
The `LabelCheck` implementation is same for containers and images and I need to return the same instance for both types.

(For now, the `ContainerCheckResult` x `ImageCheckResult` gives us no value.)